src/views/projects/Cob/AssigneeInput.svelte

```diff
@@ -75,57 +72,55 @@
   }
 </style>
 
-<div class="metadata-section">
-  <div class="header">
-    <span>Assignees</span>
-    {#if action === "edit"}
-      {#if !edit}
-        <Button
-          size="tiny"
-          variant="text"
-          on:click={() => {
-            edit = !edit;
-          }}>
-          edit
-        </Button>
-      {:else}
-        <Button
-          size="tiny"
-          variant="text"
-          on:click={() => {
-            dispatch("save", updatedAssignees);
-            edit = !edit;
-          }}>
-          save
-        </Button>
-      {/if}
-    {/if}
-  </div>
-  <div class="body">
-    {#each updatedAssignees as assignee, key (assignee)}
-      <Chip
-        on:remove={removeAssignee}
-        removeable={edit || action === "create"}
-        {key}>
-        <div class="chip-content">
-          <Avatar inline nodeId={assignee} />
-          <span>{formatNodeId(assignee)}</span>
-        </div>
-      </Chip>
+<div class="header">
+  <span>Assignees</span>
+  {#if action === "edit"}
+    {#if !edit}
```

Comment:
Can we flip this conditional around, so it's easier to read? And maybe rename the boolean to `editInProgress` to distinguish it a little bit from the action.

Reply:
78f6d26